spec/system/claims/support/schools/add_a_school_through_school_options_spec.rb

```diff
@@ -0,0 +1,128 @@
+require "rails_helper"
+
+RSpec.describe "Support User adds a School through school options", type: :system do
```

Comment:
This might be a more meaningful name to describe the test:

```suggestion
RSpec.describe "Support User adds a School without JavaScript", type: :system do
```

Reply:
@CatalinVoineag sorry – one more thing. Could you also rename the file so it matches the `describe` text?